azdev/operations/linter/linter.py

```diff
@@ -175,7 +208,8 @@ def get_linter():
                     return Linter(command_loader=command_loader, help_file_entries=help_file_entries,
                                   loaded_help=self._loaded_help)
                 return self.linter
-            self._rules[rule_type][rule_name] = rule_callable, get_linter
+
+            self._rules[rule_type][rule_name] = rule_callable, get_linter, rule_severity
```

Comment:
Do you mean `(rule_callable, get_linter, rule_severity)`?

Reply:
yes! But looks like we don't need ()